config/defaults.go

```diff
@@ -48,6 +49,9 @@ type Defaults struct {
 	// Languages specifies the default build and run container images
 	// for each known language.
 	Languages []LanguageDefault `json:"languages,omitempty"`
+
+	// KillAfter specifies the allowed time for a time to respond to a singnal
```

Comment:
typo:

```suggestion
	// KillAfter specifies the allowed time for a time to respond to a signal
```

Reply:
I rewrite this sentence, thank you for pointing this out!